File: launchdarkly/python/src/DeleteApplicationVersionExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        api.ApplicationsBetaApi(api_client).delete_application_version(
            application_key="applicationKey_string",
            version_key="versionKey_string",
        )
    except ApiException as e:
        print("Exception when calling ApplicationsBetaApi#delete_application_version: %s\n" % e)

File: launchdarkly/ruby/src/DeleteApplicationVersionExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    LaunchDarklyClient::ApplicationsBetaApi.new.delete_application_version(
        "applicationKey_string", # application_key
        "versionKey_string", # version_key
    )
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling ApplicationsBetaApi#delete_application_version: #{e}"
end
